client/common/common.c

```diff
@@ -171,6 +171,7 @@ usage(FILE *out, const char *name)
           " -F, --filter          filter entries for a given string.\n"
           " -w, --wrap            wraps cursor selection.\n"
           " -l, --list            list items vertically with the given number of lines.\n"
+          " -c, --center          list items in the center of the screen.\n"
```

Comment:
remove this as this is backend specific option

Reply:
You right ! Removed !